docs/index.md

```diff
@@ -9,7 +9,7 @@
   ![GitLabForm logo](images/gitlabform-logo.png)
 </figure>
 
-is a specialized configuration as a code tool for GitLab:
+is a specialized configuration-as-code tool for GitLab:
```

Comment:
Should we add name of the tool at the beginning? I'm thinking that probably makes it more complete sentence and also helps with accessibility, SEO, etc.

Right now you have to read the image and then the text, which seems strange.

Thoughts?